src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-command.js

```diff
@@ -34,7 +36,8 @@ class IbexaCustomAttributesCommand extends Command {
     refresh() {
         const { selection } = this.editor.model.document;
         const parentElement = selection.getSelectedElement() ?? selection.getFirstPosition().parent;
-        const { attributes, classes } = window.ibexa.richText.alloyEditor;
+        const attributes = getCustomAttributesConfig();
+        const classes = getCustomClassesConfig();
```

Comment:
```suggestion
        const customClassesConfig = getCustomClassesConfig();
```
In other places you are using `customClassesConfig` insted `classess`